Tribler/Main/tribler_main.py

```diff
@@ -290,11 +289,15 @@ def __init__(self, params, installdir, autoload_discovery=True,
             self.ready = True
 
         except Exception as e:
-            if self.splash:
-                self.splash.Destroy()
-
+            self.guiUtility.destroy_startup_splash()
             self.onError(e)
 
+    def startup_tick(self, subject, changetype, objectID, *args):
```

Comment:
Could you make those inner methods and prepend `on_` to the names as they are event callbacks?